Formula/srt.rb

```diff
@@ -22,6 +22,9 @@ def install
     openssl = Formula["openssl@1.1"]
     system "cmake", ".", "-DWITH_OPENSSL_INCLUDEDIR=#{openssl.opt_include}",
                          "-DWITH_OPENSSL_LIBDIR=#{openssl.opt_lib}",
+                         "-DCMAKE_INSTALL_BINDIR=bin",
+                         "-DCMAKE_INSTALL_LIBDIR=lib",
+                         "-DCMAKE_INSTALL_INCLUDEDIR=include",
```

Comment:
Shouldn't it be like this?

```suggestion
                         "-DCMAKE_INSTALL_BINDIR=#{opt_bin}",
                         "-DCMAKE_INSTALL_LIBDIR=#{opt_lib}",
                         "-DCMAKE_INSTALL_INCLUDEDIR=#{opt_include}",
```

Reply:
No, those macros resolve to absolute paths. The CMAKE variables being defined are relative paths used with CMAKE_INSTALL_PREFIX.

This fix is only reinforcing what should be the defaults for these variables but are lost in some interaction between a nonstandard install prefix (`/opt/homebrew`) and the "Release" target (as defined by `std_cmake_args`).

See https://cmake.org/cmake/help/latest/command/install.html?highlight=cmake_install_libdir#installing-targets